sdk/cosmosdb/cosmos/test/public/functional/query.spec.ts

```diff
@@ -266,6 +266,91 @@ describe("Queries", function (this: Suite) {
         assert.equal(sum[0], 66);
       });
     });
+
+    describe.skip("MakeList query iterator", function (this: Suite) {
```

Comment:
Just curious, how come these are skipped?

Reply:
this is waiting upon latest public emulator release.